source/grid/tria.cc

```diff
@@ -43,6 +43,40 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+
+template <int structdim>
+CellData<structdim>::CellData()
+  : material_id(0)
+  , manifold_id(numbers::flat_manifold_id)
+{
+  for (unsigned int i = 0; i < GeometryInfo<structdim>::vertices_per_cell; ++i)
+    vertices[i] = numbers::invalid_unsigned_int;
```

Comment:
You could use `std::fill` here.

Reply:
Sure. `vertices` is a C-style array, so I can't use `vertices.begin()`, but `std::begin(vertices)` works.